internal/cluster/cluster.go

```diff
@@ -1415,7 +1415,7 @@ func (m *Manager) deleteClusterFiles(ctx context.Context, c *common.Cluster, obj
 	var failedToDelete []string
 	for _, file := range files {
 		//skip log and manifests deletion when deleting cluster files
-		if folder == "" && (strings.Contains(file, "logs") || strings.Contains(file, "manifests")) {
+		if folder == "" && (strings.Contains(file, "logs") || strings.Contains(file, "manifests") || strings.Contains(file, "manifest-attributes")) {
```

Comment:
Now that we can tell apart user manifests from our own service manifests, I think it's best that this function deletes the service manifests.

The original motivation for this skip was that we didn't want to destroy use uploaded manifests when the user resets their installation, but preserving the serving generated ones is useless, as they would anyway get regenerated when the user hits install again

Reply:
Please link the ticket once you created it

Reply:
https://issues.redhat.com/browse/MGMT-15073